src/utils/feature-flags.ts

```diff
@@ -9,8 +9,9 @@ declare const window: any
  */
 const defaultFlagSet: LDFlagSet = {
   'alteration-ui-enabled': false,
-  'correction-ui-enabled': false,
-  'change-ui-enabled': false
+  'change-ui-enabled': false,
+  'conversion-ui-enabled': false,
+  'correction-ui-enabled': false
```

Comment:
These are the defaults in case the user's browser can't get the actual flags from LaunchDarkly. So it's safe.

(When running locally, we use the Dev values from LD.)